sootup.java.bytecode/src/main/java/sootup/java/bytecode/interceptors/typeresolving/BytecodeHierarchy.java

```diff
@@ -99,7 +106,8 @@ public boolean isAncestor(@Nonnull Type ancestor, @Nonnull Type child) {
           // TODO: [ms] check: the dimension condition check as it seems weird?
```

Comment:
as you are currently in the topic - does this check make sense i.e. can we remove my comment?